frontEnd/src/main.js

```diff
@@ -2,29 +2,23 @@ import Vue from 'vue'
 import App from './App.vue'
 import router from './router'
 import vuetify from './plugins/vuetify'
-import VueCookies from 'vue-cookies'
-import firebase from 'firebase'
+// import VueCookies from 'vue-cookies'
+import "./firebase"
+import { auth } from "./firebase";
 
-//var VueCookies = require('vue-cookie');
+Vue.config.productionTip = false;
```

Comment:
This could use an inline comment or something. I have no idea what this is doing.